scripts/launch_lite.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd "$(dirname ${0})/.."
+source scripts/helpers.sh
+
+pnpm_required
+
+pnpm --filter @gradio/app dev:lite
+python -m http.server -d js/lite
```

Comment:
Launching the preview HTTP server might be optional, as we see this as a build script.